src/CoreVideo/CVPixelBuffer.cs

```diff
@@ -84,6 +92,32 @@ public NSDictionary GetAttributes (NSDictionary [] attributes)
 				throw new ArgumentException (ret.ToString ());
 			return Runtime.GetNSObject<NSDictionary> (resolvedDictionaryOut);
 		}
+
+#if NET
+		[SupportedOSPlatform ("ios15.0")]
+		[SupportedOSPlatform ("tvos15.0")]
+		[SupportedOSPlatform ("macos12.0")]
+		[UnsupportedOSPlatform ("maccatalyst")] 
+		[UnsupportedOSPlatform ("watchos")]
```

Comment:
Like this?

```suggestion
		[assembly: SupportedOSPlatform ("ios15.0")]
		[assembly: SupportedOSPlatform ("tvos15.0")]
		[assembly: SupportedOSPlatform ("macos12.0")]
		[assembly: UnsupportedOSPlatform ("maccatalyst")] 
```

Reply:
no, my comment was that `watchos` is already handled with an assembly-level attribute (no need to repeat it)

Reply:
IOW those were fine
```
 		[SupportedOSPlatform ("ios15.0")]
 		[SupportedOSPlatform ("tvos15.0")]
 		[SupportedOSPlatform ("macos12.0")]
 		[UnsupportedOSPlatform ("maccatalyst")] 
```

Reply:
ahhh see, I misunderstood, got it